src/library/autodj/autodjprocessor.cpp

```diff
@@ -195,6 +188,17 @@ AutoDJProcessor::~AutoDJProcessor() {
     delete m_pAutoDJTableModel;
 }
 
+void AutoDJProcessor::slotNumberOfDecksChanged(int decks) {
+    for (int i = m_decks.size(); i < decks; ++i) {
+        BaseTrackPlayer* pPlayer = m_pPlayerManager->getDeckBase(i);
+        // Shouldn't be possible.
+        VERIFY_OR_DEBUG_ASSERT(pPlayer) {
+            continue;
```

Comment:
You should insert a null entry in `m_decks`, otherwise you may have some offset issues in `PlayerManager::getDeck`